modules/core/src/smithy4s/Hints.scala

```diff
@@ -47,6 +47,7 @@ trait Hints {
   final def has[A](implicit key: ShapeTag[A]): Boolean = this.get[A].isDefined
   final def get[A](key: ShapeTag.Has[A]): Option[A] = get(key.getTag)
   final def get[T](nt: Newtype[T]): Option[nt.Type] = get(nt.tag)
+  final def get[T](vnt: ValidatedNewtype[T]): Option[vnt.Type] = get(vnt.tag)
```

Comment:
possibly worth making a common supertype for `Newtype` and `ValidatedNewtype` to avoid the duplication of method here

Reply:
Changing this breaks binary compatibility (now that the branch is rebased on 0.18).

Reply:
Looks like it's just this one method:

```
[error] [core3]  * method get(smithy4s.Newtype)scala.Option in interface smithy4s.Hints does not have a correspondent in current version
```

can we add it back and make it package-private+deprecated, for example? That could possibly let us move forward.